service/history/execution/mutable_state_builder_methods_signal_test.go

```diff
@@ -0,0 +1,181 @@
+package execution
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/common/types"
+	"github.com/uber/cadence/service/history/constants"
+	"testing"
+)
+
+func Test__IsSignalRequested(t *testing.T) {
+	mb := testMutableStateBuilder(t)
+	requestID := "101"
+	t.Run("signal not found", func(t *testing.T) {
+		result := mb.IsSignalRequested(requestID)
+		assert.False(t, result)
+	})
+	t.Run("signal found", func(t *testing.T) {
+		mb.pendingSignalRequestedIDs[requestID] = struct{}{}
+		result := mb.IsSignalRequested(requestID)
+		assert.True(t, result)
+	})
+}
+
+func Test__GetSignalInfo(t *testing.T) {
+	mb := testMutableStateBuilder(t)
```

Comment:
Same comment as above

Reply:
yes the method is only a lookup so I thought of reusing